### DISCUSSION TITLE
Cant open web ui for influx db.

### DISCUSSION THREAD

**Discussion by tstill88** [tteck/Proxmox#761] [Help Needed] [answered]
Ive installed the script via Influx DB using v1, everything reported OK but im not able to reach the web page via http://ip:8086

Are there any steps that i need to run before i can reach the ui?

**Comment by tteck** [comment]
Where did you see InfluxDB Interface IP:8086?

**Comment by tstill88** [comment]
Port 8086 is the default port for influx normally, am I wrong?

**Comment by tteck** [comment]
I thought Versions 1.3 and later do not support the web admin interface. 🤷🏻‍♂️  I've never used InfluxDB

**Reply by tstill88** [reply]
Maybe my mistake. I've only ever used influx with a webui, I guess this is a CLI only version? If so no worries, might just go portainer route

**Reply by dimatx** [reply]
@tstill88 - wondering which route you went. it doesn't appear that this LXC has any of the UI, I'm not sure how to manage retention etc otherwise.

**Reply by tstill88** [reply]
> @tstill88 - wondering which route you went. it doesn't appear that this LXC has any of the UI, I'm not sure how to manage retention etc otherwise.

I ended up using it as an add-on in Home Assistant which is largely the data I'm working with anyway. I may revisit this another time as I'd rather have this separate to be honest.

**Comment by jeeftor** [comment]
I'm about 90% sure the install script doesn't actually start influx... once you've done that you can hit the web-ui

**Reply by dimatx** [reply]
I've been successfully feeding data into influx (so it must be running?), but can't get to the UI. What port are you able to access the UI on?

**Comment by sebasdt** [comment]
I might be late to the party but Im not sure how to enable the web interface I find it way easier to just access it and vieuw the data that way.
When looking for the influxdb v2 port 8086 I see it on IPv6 with the command here below but IPv6 isnt enabled. 
`lsof -i -P -n` shows:
![image](https://user-images.githubusercontent.com/67417070/218881715-ec8c1339-a5d1-49e2-b1b7-8a51f55d278a.png)

If possible It would greatly help me and others to enable / access the web interface.

**Reply by tteck** [reply]
Installed InfluxDB v2 using "Advanced" settings, static IP and disabling IPv6 (also disabled in router)
```
    ____      ______           ____  ____ 
   /  _/_v5  / __/ /_  ___  __/ __ \/ __ )
   / // __ \/ /_/ / / / / |/_/ / / / __  |
 _/ // / / / __/ / /_/ />  </ /_/ / /_/ / 
/___/_/ /_/_/ /_/\__,_/_/|_/_____/_____/  
 
Using Advanced Settings
Using Container Type: 1
Using Root Password: Automatic Login
Using Container ID: 112
Using Hostname: influxdb
Using Disk Size: 8
Allocated Cores: 2
Allocated RAM: 2048
Using Bridge: vmbr0
Using IP Address: 192.168.86.252/24
Using Gateway IP Address: 192.168.86.1
Disable IPv6: Yes
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server IP Address: Host
Using MAC Address: Default
Using Vlan: Default
Enable Root SSH Access: No
Enable Verbose Mode: No
Creating a InfluxDB LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using local-lvm for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 112 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.86.252 
 ✓ Internet Connected
 ✓ DNS Resolved github.com to 140.82.121.4
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Set up InfluxDB Repository
Which version of InfluxDB to install? (1 or 2) 2
 ✓ Installed InfluxDB
Would you like to add Telegraf? <y/N> n
 ✓ Customized Container
 ✓ Cleaned
 ✓ Completed Successfully!
```

`lsof -i -P -n`
```
root@influxdb:~# lsof -i -P -n
COMMAND    PID            USER   FD   TYPE   DEVICE SIZE/OFF NODE NAME
systemd      1            root   30u  IPv6 32331222      0t0  TCP *:22 (LISTEN)
systemd-r  103 systemd-resolve   11u  IPv4 32323544      0t0  UDP *:5355 
systemd-r  103 systemd-resolve   12u  IPv4 32323545      0t0  TCP *:5355 (LISTEN)
systemd-r  103 systemd-resolve   13u  IPv6 32323547      0t0  UDP *:5355 
systemd-r  103 systemd-resolve   14u  IPv6 32323548      0t0  TCP *:5355 (LISTEN)
systemd-r  103 systemd-resolve   17u  IPv4 32323550      0t0  UDP 127.0.0.53:53 
systemd-r  103 systemd-resolve   18u  IPv4 32323551      0t0  TCP 127.0.0.53:53 (LISTEN)
master     304            root   13u  IPv4 32332831      0t0  TCP 127.0.0.1:25 (LISTEN)
influxd   2751        influxdb    9u  IPv6 32340914      0t0  TCP *:8086 (LISTEN)
root@influxdb:~# 
```
Going to `192.168.86.252:8086` results in

![Screenshot 2023-02-14 6 05 08 PM](https://user-images.githubusercontent.com/21342670/218885290-124d4e0e-160a-4aa2-ac9f-64d68a8e278d.png)

**Reply by tteck** [reply]
Default settings

```
    ____      ______           ____  ____ 
   /  _/_v5  / __/ /_  ___  __/ __ \/ __ )
   / // __ \/ /_/ / / / / |/_/ / / / __  |
 _/ // / / / __/ / /_/ />  </ /_/ / /_/ / 
/___/_/ /_/_/ /_/\__,_/_/|_/_____/_____/  
 
Using Default Settings
Using Container Type: Unprivileged NO DEVICE PASSTHROUGH
Using Root Password: Automatic Login
Using Container ID: 112
Using Hostname: influxdb
Using Disk Size: 8GB
Allocated Cores 2
Allocated Ram 2048
Using Bridge: vmbr0
Using Static IP Address: dhcp
Using Gateway Address: Default
Disable IPv6: No
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server Address: Host
Using MAC Address: Default
Using VLAN Tag: Default
Enable Root SSH Access: No
Enable Verbose Mode: No
Creating a InfluxDB LXC using the above default settings
 ✓ Using local for Template Storage.
 ✓ Using local-lvm for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 112 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.86.11 
 ✓ Internet Connected
 ✓ DNS Resolved github.com to 140.82.121.3
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Set up InfluxDB Repository
Which version of InfluxDB to install? (1 or 2) 2
 ✓ Installed InfluxDB
Would you like to add Telegraf? <y/N> n
 ✓ Customized Container
 ✓ Cleaned
 ✓ Completed Successfully!
```
`lsof -i -P -n`

```
root@influxdb:~# lsof -i -P -n
COMMAND    PID            USER   FD   TYPE   DEVICE SIZE/OFF NODE NAME
systemd      1            root   45u  IPv6 32403475      0t0  TCP *:22 (LISTEN)
dhclient   109            root    9u  IPv4 32403643      0t0  UDP *:68 
systemd-r  116 systemd-resolve   11u  IPv4 32403665      0t0  UDP *:5355 
systemd-r  116 systemd-resolve   12u  IPv4 32403666      0t0  TCP *:5355 (LISTEN)
systemd-r  116 systemd-resolve   13u  IPv6 32403668      0t0  UDP *:5355 
systemd-r  116 systemd-resolve   14u  IPv6 32403669      0t0  TCP *:5355 (LISTEN)
systemd-r  116 systemd-resolve   16u  IPv4 32403671      0t0  UDP 127.0.0.53:53 
systemd-r  116 systemd-resolve   17u  IPv4 32403672      0t0  TCP 127.0.0.53:53 (LISTEN)
master     312            root   13u  IPv4 32404588      0t0  TCP 127.0.0.1:25 (LISTEN)
master     312            root   14u  IPv6 32404589      0t0  TCP [::1]:25 (LISTEN)
influxd   2769        influxdb    9u  IPv6 32413238      0t0  TCP *:8086 (LISTEN)
root@influxdb:~# 
```

`http://192.168.86.11:8086/onboarding/0`

![Screenshot 2023-02-14 6 05 08 PM](https://user-images.githubusercontent.com/21342670/218888151-bffc7f9f-7c70-449b-b9a9-c00273427603.png)

**Reply by sebasdt** [reply]
hi!
Well that was kinda weird reinstalling the container and the page is working. well maybe ipv6 wasnt disabled in the end.
Sorry for bothering and thank you for your replies!

**Reply by tteck** [reply]
No bother

**Comment by tteck** [comment, marked as answer]
InfluxDB v1.x does not have a web UI included with the software by default. Instead, it provides a command-line interface (CLI) that allows you to interact with the database and perform administrative tasks.

If you prefer a web UI for InfluxDB, it may be worth considering upgrading to InfluxDB v2.x, which includes a built-in web UI. This interface provides a range of features for working with the database, including data visualization, querying, and management tools, all accessible via a web browser.
